src/main/java/step2/Main.java

```diff
@@ -0,0 +1,41 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+        int money = InputView.getMoney();
+
+        Generator lottoGenerator = new LottoGenerator();
+        lottoGenerator.setNumber("");
+
+        Lottos lottos = new Lottos(money, lottoGenerator);
+
+        ResultView.printLottoCnt(lottos.getLottoList().size());
```

Comment:
getter를 이용하는것보다 바로 size를 반환해주는 메서드를 만들면 어떨까요 :) ?
```suggestion
        ResultView.printLottoCnt(lottos.size());
```